extensions/stackdriver/metric/record.cc

```diff
@@ -169,15 +169,63 @@ TagKeyValueList getInboundTagMap(
   return inboundMap;
 }
 
+// See:
+// https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto
+uint32_t httpCodeFromGrpc(uint32_t grpc_status) {
+  switch (grpc_status) {
+    case 0:
```

Comment:
Can we add the status code names here?

Reply:
do you mean in the code or in additional comments?

all we have is the uint32_t (in RequestInfo) unfortunately. we could convert and then convert again, but that seems a bit wasteful.

Reply:
sorry meant just in comments....

Reply:
Done.